### PR TITLE
Get a diff between old and new content data files

### PR DESCRIPTION
**Pull request by genadyp**
1.code moved from lib to bin 
2.file_utils installed in the system along with other executables 
3. output to STDOUT added
4. add parameter check for ContentData.to_file method
5.help added

Proposed solution for #246

### CODE REVIEW

**Comment by bbfsdev** [comment]
Review done.

File: bin/file_utils [hunk 1]
```diff
@@ -2,11 +2,119 @@
...
 
-require 'file_utils'
+require 'content_data'
 require 'log'
 require 'params'
 
+#############################################
+# Definitions
+#############################################
+Params.string 'command', nil ,'path'
```
Comment (bbfsdev) [inline bin/file_utils]: Why command is a path?

File: bin/file_utils [hunk 2]
```diff
@@ -2,11 +2,119 @@
...
+Usage:
+  To get merge of two ContentData files,
+  i.e. contents and appropreate instances that exist in at least one ContentData file:
+    file_utils --command=merge --cd_a=<content_data_file1> --cd_b=<content_data_file_2> [--dest=<result_content_data_file>]
+  To get intersection of two ContentData files,
+  i.e. contents and appropreate instances that exist in the both ContentData files:
+    file_utils --command=intersect --cd_a=<content_data_file1> --cd_b=<content_data_file_2> [--dest=<result_content_data_file>]
+  To get content_data_2 minus content_data_1,
+  i.e. contents and appropreate instances that exist in content_data_file2 and absent in content_data_file1:
+    file_utils --command=minus --cd_a=<content_data_file1> --cd_b=<content_data_file_2> [--dest=<result_content_data_file>]
```
Comment (bbfsdev) [inline bin/file_utils:29]: consider adding diff command (not symmetric)
it is intersect, then minus.

File: bin/file_utils [hunk 3]
```diff
@@ -2,11 +2,119 @@
...
 Log.init
 
-FileUtils::FileUtils.run
+#############################################
+# Methods
+#############################################
+
+# Algebra of sets operations (merge, intersect, minus) on ContentData files
+def content_data_command()
+  if Params['cd_a'].nil?
```
Comment (bbfsdev) [inline bin/file_utils]: Isn't Params have a required field? Please use it and remove those lines.

File: bin/file_utils [hunk 4]
```diff
@@ -2,11 +2,119 @@
...
+  rescue Exception => e
+    err_msg = "Error loading content data cd_a = %s: %s" % [Params['cd_a'], e.message]
+    puts err_msg
+    Log.error(err_msg)
+    Log.flush
+    return
+  end
+
+  if Params['cd_b'].nil?
+    err_msg = "--cd_b is not set"
```
Comment (bbfsdev) [inline bin/file_utils]: This code repeats itself.

**Comment by bbfsdev** [comment]
Finished review

File: bin/file_utils (hunk 1 above)
Comment (genadyp) [inline bin/file_utils]: fixed

File: bin/file_utils (hunk 2 above)
Comment (genadyp) [inline bin/file_utils:29]: Is it make sense? need a discussion.

File: bin/file_utils (hunk 3 above)
Comment (genadyp) [inline bin/file_utils]: we check here that a param was set

File: bin/file_utils (hunk 4 above)
Comment (genadyp) [inline bin/file_utils]: I know :) Isn't more readable in this case?

**Pull request merged**